### PR TITLE
Reference bittner/lego-mindstorms-ev3-comparison

### PR DESCRIPTION
**Pull request by ddemidov**

### CODE REVIEW

**Comment by ddemidov** [comment]
See https://gitter.im/ev3dev/chat?at=5a4aa25d84fb74b940279449

File: README.md [hunk 1]
```diff
@@ -71,3 +71,7 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## Differences between Home and Education sets.
```
Comment (bittner) [inline README.md]: I think the trailing period can be omitted.

File: README.md [hunk 2]
```diff
@@ -71,3 +71,7 @@ class with child classes for the various permutations of TRACK3R
...
+
+There is a nice project at [bittner/lego-mindstorms-ev3-comparison](https://github.com/bittner/lego-mindstorms-ev3-comparison) highlighting differences LEGO MINDSTORMS EV3 Home Edition (#31313) and LEGO MINDSTORMS Education EV3 (#45544).
```
Comment (bittner) [inline README.md]: ... also referencing the robot programs from this repository and a few others.

(Could maybe be added?)

Comment (ddemidov) [reply README.md]: Done in 41c6f42

File: README.md [hunk 3]
```diff
@@ -71,3 +71,13 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## Differences between Home and Education sets
```
Comment (WasabiFan) [inline README.md]: Mention robot programs in the title?

File: README.md [hunk 4]
```diff
@@ -71,3 +71,13 @@ class with child classes for the various permutations of TRACK3R
...
+
+There is a nice project at
+[bittner/lego-mindstorms-ev3-comparison](https://github.com/bittner/lego-mindstorms-ev3-comparison)
+highlighting differences between LEGO MINDSTORMS EV3 Home Edition (#31313) and LEGO
```
Comment (WasabiFan) [inline README.md]: I think we should put code formatting around numbers so that GitHub doesn't make them links :laughing:

**Review by WasabiFan — APPROVED** [review]

**Comment by WasabiFan** [comment]
Side-note: I'm planning to carve out some time over the next few days to finish up my open PRs and maybe get Micropython figured out, time permitting.

File: README.md (hunk 4 above)
Comment (ddemidov) [reply README.md]: It will be a while before github will be able to link to those issues, but I did it anyway :)

File: README.md (hunk 3 above)
Comment (ddemidov) [reply README.md]: I can not think of a nice way to do this. Any ideas?

Comment (WasabiFan) [reply README.md]: Hmm... I don't have any either :grinning: What about putting it under a separate heading, perhaps "3rd-party index of demo robots"?

Comment (WasabiFan) [reply README.md]: It's nothing major; I'm happy with whatever you think is reasonable.

Comment (ddemidov) [reply README.md]: The new section is just two sentences, and if we mention robots in the section header, the header becomes as informative as the whole section. So, unless @bittner has some other ideas, I would leave the header as is.

Comment (bittner) [inline README.md]: What would you think about just focussing on "more demo programs" and omitting the mentioning of the comparison of "Home Edition" and "Education Edition"? I don't think that users or developers coming to this README primarily care about the difference between the two hardware sets. They may much more care about "more robot programs" though.

Hence, the paragraph may look like instead:
> ## More Robot Programs
> 
> The [LEGO Mindstorms EV3 Comparison project](https://github.com/bittner/lego-mindstorms-ev3-comparison) sports a few other robot programs in addition. Its [comparison table](https://github.com/bittner/lego-mindstorms-ev3-comparison#inspiration-for-lego-ev3-robots) also lists programs created for other platforms than ev3dev.

What do you guys think?

**Comment by WasabiFan** [comment]
@ddemidov feel free to merge whenever you are happy with it; I have no reservations.

File: README.md [hunk 5]
```diff
@@ -71,3 +71,11 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## More robot programs
+
+The [LEGO Mindstorms EV3 Comparison
+project](https://github.com/bittner/lego-mindstorms-ev3-comparison) sports a
+few other robot programs in addition. Its [comparison
+table](https://github.com/bittner/lego-mindstorms-ev3-comparison#inspiration-for-lego-ev3-robots)
```
Comment (bittner) [inline README.md:80]: Note that if you prefer shorter lines of code you can also cut off links in MarkDown after the opening brace, e.g.
```
Its [comparison table](
https://...very...very...very...loooooooooooog...url/...)
also lists ...
```

**Comment by ddemidov** [comment]
@WasabiFan, I don't have merge rights here :)

**Comment by WasabiFan** [comment]
Uhh... That's odd. I don't have admin rights so can't manage permissions, but I had thought this repo had @ev3dev/python on the "write access" list. @dlech?

**Comment by dlech** [comment]
Right now, write permissions are managed on an individual basis.

**Comment by WasabiFan** [comment]
Can we at least add @ddemidov?

**Comment by dlech** [comment]
done

**Pull request merged**